repo/format/format_blob_cache.go

```diff
@@ -12,133 +14,161 @@ import (
 	"github.com/kopia/kopia/internal/cache"
 	"github.com/kopia/kopia/internal/cachedir"
 	"github.com/kopia/kopia/internal/clock"
-	"github.com/kopia/kopia/internal/gather"
 	"github.com/kopia/kopia/repo/blob"
-	"github.com/kopia/kopia/repo/logging"
 )
 
 // DefaultRepositoryBlobCacheDuration is the duration for which we treat cached kopia.repository
 // as valid.
 const DefaultRepositoryBlobCacheDuration = 15 * time.Minute
 
-var log = logging.Module("kopia/repo/format")
+// BlobCache encapsulates cache for format blobs.
+type BlobCache interface {
```

Comment:
Is it necessary to include the BlobCache in this PR?  If so, why?

Reply:
same, it's only for 2 blobs.

Reply:
added comment, made this unexported.